frontend/src/metabase/collections/components/CollectionEmptyState/CollectionEmptyState.tsx

```diff
@@ -60,25 +59,32 @@ const DefaultCollectionEmptyState = ({
   const canWrite = !!collection?.can_write;
 
   return (
-    <EmptyStateRoot data-testid="collection-empty-state">
+    <EmptyStateWrapper>
       <CollectionEmptyIcon />
       <EmptyStateTitle>{t`This collection is empty`}</EmptyStateTitle>
-      <Text size="1rem" color="text-medium" align="center" mb="1.5rem">
+      <EmptyStateSubtitle>
         {t`Use collections to organize and group dashboards and questions for your team or yourself`}
-      </Text>
+      </EmptyStateSubtitle>
       {canWrite && (
         <NewItemMenu
           trigger={<Button icon="add">{t`Create a new…`}</Button>}
           collectionId={collection?.id}
         />
       )}
-    </EmptyStateRoot>
+    </EmptyStateWrapper>
   );
 };
 
 const CollectionEmptyIcon = (): JSX.Element => {
+  const theme = useMantineTheme();
   return (
-    <svg width="117" height="94" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <svg
+      viewBox="0 0 117 94"
+      width={theme.other.collectionBrowser.emptyContent.icon.width}
+      height={theme.other.collectionBrowser.emptyContent.icon.height}
```

Comment:
nit: we could destructure out `emptyContent` if you want to shorten this. same as the other 6 occurrences below

Reply:
I'm writing a quick utility to get theme keys so I can just move that to this PR

Reply:
ok I'll put it in a different PR. typescript is being difficult